### ISSUE TITLE
Different support number with data when evaluation (classification_report in LayoutLMv2)

### ISSUE THREAD

**Issue by yellowjs0304** [NielsRogge/Transformers-Tutorials#137]
Hi, @NielsRogge  It's really confused result for me.
First, i used your [code ](https://github.com/NielsRogge/Transformers-Tutorials/blob/master/LayoutLMv2/CORD/Fine_tuning_LayoutLMv2ForTokenClassification_on_CORD.ipynb)with my own dataset like CORD dataset, and I don't used BIO tag each labels.

While analyzing the training results, I found the classification_report (from seqeval.metrics that you used) function returns diffrent support number. I know as "support" means the number of each class data. Is this wrong?

Please see below example.
![image](https://user-images.githubusercontent.com/30281608/176134822-8fc581df-4fd4-4896-8307-ece861e0ccf4.png)

It looks like the support number is different with gt_label lists.
Am i wrong?? Could you please share any information if you know.

**Comment by LucaMalagutti** [comment]
The reason for this behavior is that seqeval's classification_report expects IOB-style labels as input [(see this issue)](https://github.com/chakki-works/seqeval/issues/75), which is also why all your labels are missing their initial letter when printed in the classification report.

Afaik, as a workaround, you can either:
1. Add a "B" as the initial letter of every label in your lists
2. Flatten your lists and use the similar [sklearn's classification_report](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html), which does not have this problem

**Comment by yellowjs0304** [comment]
Thank you. I understood. As you told, I added "B-" token each of labels, and Now the results is great. 
 And the CORD notebook code doesn't have any code related this issue. I think it must be describe in notebook code, also. 
 @NielsRogge
